ansible/module_utils/port_utils.py

```diff
@@ -130,6 +130,9 @@ def get_port_alias_to_name_map(hwsku, asic_name=None):
                     port_alias_to_name_map["etp%d%s" % (i, j)] = "Ethernet%d" % ((i - 1) * 8 + x - 1)
             port_alias_to_name_map["etp65"] = "Ethernet512"
             port_alias_to_name_map["etp66"] = "Ethernet513"
+        elif hwsku == "Arista-7060X6-64PE-B-P64":
+            for i in range(1, 65):
+                port_alias_to_name_map["etp%d" % (i)] = "Ethernet%d" % ((i - 1) * 8)
```

Comment:
hi @bingwang-ms  for this one should we merge it with line 111

Reply:
Fixed. Thanks!